image/tarexport/save.go

```diff
@@ -288,10 +288,11 @@ func (s *saveSession) saveImage(id image.ID) (map[layer.DiffID]distribution.Desc
 	var layers []string
 	var foreignSrcs map[layer.DiffID]distribution.Descriptor
 	for i := range img.RootFS.DiffIDs {
+		v1ImgCreated := time.Unix(0, 0)
 		v1Img := image.V1Image{
 			// This is for backward compatibility used for
 			// pre v1.9 docker.
-			Created: time.Unix(0, 0),
+			Created: &v1ImgCreated,
 		}
```

Comment:
I'll create a tracking ticket for this; if all this handling is to allow  `docker export` to save an image that can be imported on a v1.8 version of Docker, then I think we should consider ditching all that stuff. If someone is running docker 1.8 in 2023, they have a big problem 😂.